sunpy/io/tests/test_jp2.py

```diff
@@ -22,6 +23,15 @@ def test_read_header():
     header = jp2.get_header(AIA_193_JP2)[0]
     assert isinstance(header, FileHeader)
 
+@skip_glymur
+def test_read_header_comments():
+    header = jp2.get_header(EUI_174_JP2)[0]
+    assert len(header["KEYCOMMENTS"].items()) == 222
+
+@skip_glymur
+def test_read_header_history():
+    header = jp2.get_header(EUI_174_JP2)[0]
+    assert len(header["HISTORY"]) == 858
```

Comment:
Can we check some parts of the values?

Reply:
Since the string, I checked for one of the comments inside the `HISTORY` element for a substring